### PR TITLE
Add string coercion for parent

### PR DESCRIPTION
**Pull request by islean**
## Description

Addresses #2569 wherein an extra coercion to string is necessary.

### Fixed

- String coercion validator to Individuals.father and Individuals.mother.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b add-coercion-to-string-for-parent -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: tests/apps/scout/test_scout_models.py [hunk 1]
```diff
@@ -20,7 +20,30 @@ def test_validate_case_father_none(none_case_raw: dict):
...
+        RelationshipStatus.HAS_NO_PARENT.value
+    )
+
+    # THEN assert that '_id' has been changed to 'id'
+    assert "_id" not in case_dict
+    assert "id" in case_dict
+
+
+def test_validate_case_father_int(none_case_raw: dict):
+    """Test to validate string coercion when the Father is set to 0"""
```
Comment (henrikstranneheim) [inline tests/apps/scout/test_scout_models.py:33]:
```suggestion
    """Test to validate string coercion when the father is set to 0."""
```

**Review by henrikstranneheim — APPROVED** [review]
👍

**Review by diitaz93 — APPROVED** [review]
👍

**Comment by islean** [comment]
Tested on stage with a responsive CLI.

**Comment by islean** [comment]
Deployed by IO 2023-10-09:

```
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/isak.ohlsson
done.
Log deploy... done.
cg, version 51.5.5
θ61° 1d [isak.ohlsson@hasta:~] [P_base] 3m32s $ 
```

**Pull request merged**